lib/rubocop/cop/naming/inclusive_language.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+# TODO
+# - message for file names
+# - detect in file path? or basename
+# - cleanup code
+
+module RuboCop
+  module Cop
+    module Naming
+      # This cop recommends the use of inclusive language instead of problematic terms.
```

Comment:
It'd be good to explain here the scope of the cop - what exactly does it check for (identiers, strings, comments, etcs).